code/stable/dblpendulum/TraceyGraph/allvsall.dot

```diff
@@ -0,0 +1,94 @@
+digraph allvsall {
```

Comment:
Now, here's a good question: do we want these in stable? I don't know!

Reply:
Yea, I wasn't too sure about that either. I don't think it should be much of a bother (the traceability matrices barely change, so I'm hoping the same goes for the tracey graphs once everything is smoothed out). 

Although it could be a decent indicator of something changing unexpectedly if a lot of the directions change in the graphs. And with `make stabilize`, it's not that bad to compare and copy files.

It might also be nice to implement the graphs directly into the SRS, though I'm not too sure how that would be done. Maybe we could generate .png files for them and then include them into the SRS files as a figure? It could be its own section, just as the traceability matrix has its own section.

Reply:
Generating the .png and including a reference to them in the SRS sounds like a good idea. The .png themselves should not be compared when doing a diff (that they exist should be checked).

Let's experiment with these being part of stable. We'll see if we like it.

Reply:
Okay, I think I'll do a bit of cleanup in terms of labels before merging this PR, and then start including the .png files in a new PR. We could also use pdfs similar to the module dependency graphs on the drasil website, where we just link the traceability graphs.